modules/development.nix

```diff
@@ -7,8 +7,8 @@ with import ./../lib.nix;
     global = {
       allocateElasticIP            = false;
       enableEkgWeb                 = true;
-      dnsHostname                  = null;
-      dnsDomainname                = mkForce null;
+      dnsHostname                  = "${name}-${config.deployment.name}";
```

Comment:
`name` has the same value as `config.deployment.name`, did you verify the resulting URL?

Reply:
@domenkozar, are you sure?

I'm reading https://nixos.org/nixops/manual/#ssec-relnotes-1.3 differently.

Also: https://github.com/NixOS/nixops/blob/master/nix/eval-machine-info.nix#L73

Reply:
As to the DNS provisioning -- I've tested it, indeed.

Reply:
You're right, `name` is machine name.